test/tests.ts

```diff
@@ -0,0 +1,115 @@
+import jwt_decode, { InvalidTokenError, JwtPayload } from "./../lib/index";
+import { expect } from "@jest/globals";
+
+var token =
+  "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmb28iOiJiYXIiLCJleHAiOjEzOTMyODY4OTMsImlhdCI6MTM5MzI2ODg5M30.4-iaDojEVl0pJQMjrbM1EzUIfAZgsbK_kgnVyVxFSVo";
+
+describe("jwt-decode", function () {
+  it("should fail to construct without a clientID", function () {
```

Comment:
Should this test be asserting an error?

Reply:
Thanks for catching this. This test has been here since a couple of years. I will rename it